src/lime/protocol/identity.py

```diff
@@ -0,0 +1,65 @@
+from .constants import CommonConstants
+
+INSTANCE_SEPARATOR = '/'
```

Comment:
This should be in `CommonConstants` since it's a constant that can be used by others when referring to `Nodes`